pkgs/tools/misc/elfcat/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "elfcat";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "ruslashev";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-s56FyRoD2IhgdwEV63jMaB265CodHUlvmrWzmXAmonY=";
+  };
+
+  cargoLock.lockFile = "${src}/Cargo.lock";
```

Comment:
You need to commit this file to make it work :)

Reply:
Well I tried, doesn't work here apparently :P